ui/src/List/DisplaySelector.tsx

```diff
@@ -33,15 +33,14 @@ export const DEFAULT_DURATION = DURATIONS[0][1];
  * Returns a card for setting options relating to how videos are displayed.
  */
 const DisplaySelector = (props: Props) => {
-  const theme = useTheme();
   return (
     <Card
       sx={{
-        padding: theme.spacing(1),
         display: "flex",
         flexDirection: "column",
       }}
     >
+      <CardContent>
```

Comment:
Ahh, I guess missing this was why the `padding: theme.spacing(1)` was necessary before above?

Could you fix the indentation within this element?

Reply:
We don't need padding for this because the `<CardContent />` has done it for us. See full `Card` usage in the [MUI Documentation](https://mui.com/material-ui/react-card/#basics).

Reply:
So from the docs, looks like you're totally right that `Card` should have a `CardContent` within it.

But I noticed this morning that it spaces out the UI enough to introduce a vertical scroll bar even on my large desktop monitor; `Card` before vs `CardContent` after:

<img width="208" alt="image" src="https://github.com/scottlamb/moonfire-nvr/assets/8949524/bf3a9e2c-c867-4416-82a2-a799ef354e2b"> <img width="201" alt="image" src="https://github.com/scottlamb/moonfire-nvr/assets/8949524/530cc149-f258-48a3-9c45-9baa86c023bc">

* 12 extra pixels of interior width, I think because one or more of the cards in the column no longer fits quite as well in the column
* 8+8 => 16+16 pixels of horizontal padding
* 8+8 => 16+24 pixels of vertical padding
* margin went away, but I think that just turned into the same space showing up in a different way between elements

Looks like the padding comes from here:

https://github.com/mui/material-ui/blob/9ef12dc81c54566eeceae16bcf026a75fbce504d/packages/mui-material/src/CardContent/CardContent.js#L21-L30

All the extra spacing might be my biggest complaint in general about the mui library, e.g. <https://github.com/mui/material-ui/issues/27700>.

Looks like I could re-introduce the `sx={{ padding: theme.spacing(1) }}` on the `CardContent` and get it closer to before, but I'm not sure how to best undo the `last-child` bit.

I don't really remember why I picked `Card` to begin with, though. I just wanted something to group like controls but my usage doesn't really feel like their examples. Should it be just `Paper` instead? `<Paper sx={{ padding: theme.spacing(1) }}>` seems to render as before (with the margin going somewhere else I guess):

<img width="195" alt="image" src="https://github.com/scottlamb/moonfire-nvr/assets/8949524/bb261f62-5a96-4083-95a3-ffbe97f63a8b">

Reply:
Hmm, I think that padding at `last-child` should separate between `CardContent` and `CardActions` (see [Material Design 2 Guidelines](https://m2.material.io/components/cards#specs)) but we don't have a plan to use `CardActions`. Maybe we can override it by adding `sx={{padding: 1}}`

Reply:
With `sx={{padding: 1}}` the `last-child` thing is still there. iiuc it's more selective so it wins out.

<img width="690" alt="image" src="https://github.com/scottlamb/moonfire-nvr/assets/8949524/be4e062f-72b5-40cf-b952-2652b84ee2ec">

> 12 extra pixels of interior width, I think because one or more of the cards in the column no longer fits quite as well in the column

Huh. By commenting stuff out, I narrowed it down to the "Trim start and end" in the display selector. And it's fine if I add the enclosing `FormGroup` that it probably should have anyway.

Reply:
I found another solution, just add `CardActionArea` outside `CardContent`. Example can be found [here](https://mui.com/material-ui/react-card/#primary-action)

Reply:
So, I tried to add override `CardContent` style by adding some props to `extendTheme` in `index.tsx`.

```ts
const themeExtended = experimental_extendTheme({
  colorSchemes: {
    dark: {
      palette: {
        primary: {
          main: "#000000",
        },
        secondary: {
          main: "#e65100",
        },
      },
    },
  },
  // this...
  components: {
    MuiCardContent: {
      styleOverrides: {
        root: {
          padding: '16px',
          paddingBottom: '16px !important',
        }
      }
    }
  }
});
```

Look better now:

![image](https://github.com/scottlamb/moonfire-nvr/assets/80969068/f8ff1aa1-6c22-4eee-99bf-bdd0f1eb4483)